janusgraph-test/src/test/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java

```diff
@@ -51,35 +45,34 @@ public void graphShouldOpenWithSameInstanceId() {
         final StandardJanusGraph graph1 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
 
         assertEquals(graph1.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");
+        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");
 
         final StandardJanusGraph graph2 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
 
         assertEquals(graph1.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");
+        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");
         assertEquals(graph2.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph2.openManagement().getOpenInstances().toArray()[0], "not-unique");
+        assertEquals(graph2.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");
         graph1.close();
         graph2.close();
     }
 
     @Test
+    @Disabled("This should be run against a backend which support multiple backends.")
     public void graphShouldNotOpenWithSameInstanceId() {
         final Map<String, Object> map = new HashMap<String, Object>();
         map.put(STORAGE_BACKEND.toStringWithoutRoot(), "inmemory");
         map.put(UNIQUE_INSTANCE_ID.toStringWithoutRoot(), "not-unique");
         final MapConfiguration config = new MapConfiguration(map);
-        final StandardJanusGraph graph1 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
-
-        assertEquals(graph1.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");
-
-        thrown.expect(JanusGraphException.class);
-        final String err = "A JanusGraph graph with the same instance id [not-unique] is already open. Might required forced shutdown.";
-        thrown.expectMessage(equalTo(err));
-        final StandardJanusGraph graph2 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
-
-        graph1.close();
+        JanusGraphException janusGraphException = assertThrows(JanusGraphException.class, () -> {
+            final StandardJanusGraph graph1 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
+
+            assertEquals(graph1.openManagement().getOpenInstances().size(), 1);
+            assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");
```

Comment:
This assert fails because you added `(current)` to `not-unique` string. Remove `(current)` suffix.

Reply:
The assert didn't fail because you disabled this test. Please, enable it.

Reply:
Please change the line:
`assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");`
to
`assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");`